config/routes.rb

```diff
@@ -1,11 +1,9 @@
 Rails.application.routes.draw do
+  resources :recipes, only: [:index, :show]
+
   get 'ingredients/index'
 
   root 'ingredients#index'
 
-  get 'recipes/show'
```

Comment:
Why the  line 6 was deleted? Isn't this 'get' necessary to display the view of one chosen recipe? Just asking :P